app/components/TableWrapper.tsx

```diff
@@ -20,6 +20,7 @@ interface TableProps<T extends Record<string, unknown>> {
   columns: ColDataCardDef<T>[];
   initialQueryConfig: QueryConfig;
   localStorageKey: string;
+  contextFilters?: Filter[] 
 }
 
 /**
```

Comment:
what is the general opinion of docstrings in our new VZ typescript codebase? i think it would be helpful to have descriptions for these params here. here are the official [typescript doc specifications](https://tsdoc.org/)